tests/fuzz/CMakeLists.txt

```diff
@@ -19,7 +17,14 @@ file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/fuzz_input)
 set(TEST_CASE "pet"  CACHE STRING  "Test case (pet or manifest12)")
 
 if (${TEST_CASE} STREQUAL pet)
-    target_cddl_source(fuzz_target ../cases/pet.cddl ENTRY_TYPES Pet DECODE)
+    execute_process(
```

Comment:
The fuzz_input files are not used at all during building. They are passed to afl when running the fuzzing, after the app has been built. Because of this, there is no dependency relationship that can be expressed here, so keeping it as execute_process seems a simple as anything at this point. If we were to perform the actual fuzzing in Cmake at some point, I agree we should look into using custom commands.